v2/emailpassword/sidebars.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     "architecture",
     {
       type: 'category',
-      label: 'Quick setup',
+      label: 'Initialization',
```

Comment:
I think there are a bunch of places where we say checkout quick setup section which now needs to change